build.gradle.kts

```diff
@@ -5,7 +5,8 @@ plugins {
     id("com.github.johnrengelman.shadow")
     id("biz.aQute.bnd.builder")
     id("maven-publish")
-    id("com.jfrog.bintray")
```

Comment:
please remove the `import java.util.*` in line 1, it was only used by the bintray plugin (as far as I can remember)